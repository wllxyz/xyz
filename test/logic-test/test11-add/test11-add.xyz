<i>-->""==><o>-->($ADD,($ADD, "10", "123.45"),"HELLO","123","45");
