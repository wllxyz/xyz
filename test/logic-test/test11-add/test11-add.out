133.449997HELLO12345
